### ISSUE TITLE
Shaman - healing stream totem not working properly

### ISSUE THREAD

**Issue by phrenorrhagia** [SunstriderEmu/BugTracker#3448]
**Describe the bug**
Healing stream totem does not scale with bonus healing.
**To Reproduce**
Cast healing stream totem with a resto shaman with some bonus healing. It heals for a strikingly low value. 
1. Cast Healing stream totem
2. Observe the healing
3. Induce a conclusion

Spell: https://wowwiki.fandom.com/wiki/Healing_Stream_Totem (all the necessary info is here, wowhead does not help)

**Expected behavior**
Healing stream totem healing for more than 24 (with over 1k bonus healing) (interestingly, it heals for a little bit more in enha talents, regardless of the Restorative totems talent) 

**Screenshots/videos**

![image](https://user-images.githubusercontent.com/60427498/82453582-014e0280-9ab1-11ea-96de-f2f0a5027ab7.png)

**Comment by Jerrythemagicdog** [comment]
https://github.com/SunstriderEmu/BugTracker/issues/2606

The scaling should be 4.5% according to this table - https://wowwiki.fandom.com/wiki/Spell_power_coefficient?oldid=1492745

**Comment by Godmatik** [comment]
@phrenorrhagia 
healing stream sucks in TBC, not much you can do about it

**Comment by phrenorrhagia** [comment]
@Godmatik what an attitude, sux even more, when doesn't work as intended

**Comment by BasedForfrost** [comment]
So I just tested out the healing from Healing Stream Totem and found something interesting. With Healing Stream Totem Rank VI, the 4.5% scaling coefficient seems to be working fine. However, I noticed in your picture that you were using Rank 5, so when I dropped Healing Stream V, I had zero bonus healing.

Currently spec'd Enhancement went I tried it. I have 97 Bonus Healing with zero gear on. 
Healing Stream Rank VI (97 healing) - 22 per tick
Healing Stream Rank VI (1220 healing) - 72 per tick
Healing Stream Rank V (97 healing) - 14 per tick
Healing Stream Rank V (1220 healing) - 14 per tick

Ranks I through IV all have the same behavior. Rank IV has 12 healing per tick regardless of +healing, III has 10 healing per tick, so on and so forth.

Rank VI is the only rank of the totem received the spell power coefficient.
